torch/csrc/autograd/init.cpp

```diff
@@ -238,6 +239,9 @@ PyObject* THPAutograd_initExtension(PyObject* _unused, PyObject *unused) {
     at::clearCallbacks();
   });
 
+  py::class_<c10::InferenceMode>(_C_m, "InferenceMode")
```

Comment:
Doesn't this automatically leak into the `torch.` python namespace? You want to name it `_InferenceMode` I think.